Begrippenlijst.md

```diff
@@ -1,32 +1,58 @@
 # Begrippenlijst
-API = Een middel om programma’s met elkaar te kunnen laten communiceren.
-<br><br>
-Functioneel Programmeren = Een model dat zich richt op een hoger niveau van denken. Dit model doet berekeningen door middel van functies, het enige wat er toe doet is dat het antwoord goed is.
-<br><br>
-Markdownfiles = md files die direct aangepast kunnen worden via GitHub.
-<br><br>
-Diffs = Verschillen tussen bestanden.
-<br><br>
-Package Manager = Een package manager is een software tool die het installeren, upgraden, configureren en verwijderen van software op de pc automatiseert.
-<br><br>
-pip = pip is de <i>package manager</i> van Python.
-<br><br>
-decorator = Een decorator is een ontwerppatroon voor objectorientatie.
-<br><br>
-Template engine = Dit is een manier om data in verschillende vormen te laten zien.
-<br><br>
-Jinja 2 = Dit is de <i>template engine</i> die Python in HTML zet.
-<br><br>
-Json = Dit is een <i>template engine</i> die Python omschrijft naar Javascript.
-<br><br>
-Websocket = 
-<br><br>
-dictionairy python = 
-<br><br>
-list python = 
-<br><br>
-interpreter system = Je kan de code uitvoeren zodra het is geschreven, dit maakt testen snel.
-<br><br>
-Indentation = De spatie aan het begin van de regels
-<br><br>
+
+#### API
+Een middel om programma’s met elkaar te kunnen laten communiceren.
+
+
+#### Functioneel Programmeren
+Een model dat zich richt op een hoger niveau van denken. Dit model doet berekeningen door middel van functies, het enige wat er toe doet is dat het antwoord goed is.
+
+
+#### Markdownfiles (.md)
+md files die direct aangepast kunnen worden via GitHub.
+
+#### Diffs
+Verschillen tussen bestanden.
+
+
+### Package Manager
+Een package manager is een software tool die het installeren, upgraden, configureren en verwijderen van software op de pc automatiseert.
+
+
+### pip
+pip is de <i>package manager</i> van Python.
+
+
+### decorator 
+Een decorator is een ontwerppatroon voor objectorientatie.
+
+
+### Template engine
+Dit is een manier om data in verschillende vormen te laten zien.
+
+
+### Jinja 2
+Dit is de *template engine* die Python in HTML zet.
+
+
+### Json
+Dit is een *template engine* die Python omschrijft naar Javascript.
```

Comment:
Dit klopt niet. Json is een bestandsuitwisselingsformaat. Zie ook https://en.wikipedia.org/wiki/JSON